samples/README.md

```diff
@@ -2,324 +2,215 @@
 [//]: # "To regenerate it, use `npm run generate-scaffolding`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Storage: Node.js Samples
+# [Google Cloud Storage: Node.js Samples](https://github.com/googleapis/nodejs-storage)
 
 [![Open in Cloud Shell][shell_img]][shell_link]
```

Comment:
So I should have asked this elsewhere - why would I want to open the README in cloud shell?  I kinda think maybe that doesn't belong there

![image](https://user-images.githubusercontent.com/534619/55120220-40054a00-50b2-11e9-9993-482327cd6416.png)

Reply:
I don't know? ported this over from the old file; doesn't seem that useful to me.

Reply:
I wonder if it's supposed to open the *directory* in cloud shell?  Maybe something got lost in translation?  Because I could see opening `/samples` being useful. I say either make it do that, or we drop it